plugins/experimental/slice/client.cc

```diff
@@ -119,15 +126,30 @@ handle_client_req(TSCont contp, TSEvent event, Data *const data)
 void
 handle_client_resp(TSCont contp, TSEvent event, Data *const data)
 {
-  DEBUG_LOG("%p handle_client_resp %s", data, TSHttpEventNameLookup(event));
+  // DEBUG_LOG("%p handle_client_resp %s", data, TSHttpEventNameLookup(event));
```

Comment:
can this be removed if it's not needed?

Reply:
When troubleshooting the TSVIO's these are nice to have available.